src/inmanta/module.py

```diff
@@ -1357,23 +1409,46 @@ def get_all_submodules(self) -> List[str]:
 
         return modules
 
+    @abstractmethod
+    def _get_plugin_dir(self) -> Optional[str]:
+        """
+        Return directory containing the python files which define handlers and plugins.
+        If no such directory is defined, this method returns None.
+        """
+        raise NotImplementedError()
+
+    @abstractmethod
+    def _get_model_dir(self) -> str:
+        """
+        Return a path to the model directory of the module.
+        """
+        raise NotImplementedError()
+
+    # This method is not part of core's stable API but it is currently used by pytest-inmanta (inmanta/pytest-inmanta#76)
+    def _get_fq_mod_name_for_py_file(self, py_file) -> str:
+        """
+        Returns the fully qualified Python module name for a path to a python file in the plugins directory.
+
+        :param py_file: Absolute path to the python file in the plugins directory of the module.
```

Comment:
Indeed. It's used by `pytest-inmanta`. I assume it didn't because `_get_fq_mod_name_for_py_file` is a private method, but it does.

Reply:
Yes, it's commented above, but that can of course easily be missed. `pytest-inmanta` really shouldn't use it but it does. We can just pick up inmanta/pytest-inmanta#76 (should be a very easy fix) if we think it's all right for the next pytest-inmanta to be incompatible with pre 2020.4 releases (all latest iso versions are compatible).
Another consequence of this change then would be that only the latest (with the inmanta/pytest-inmanta#76 fix in it) would be compatible with the next OSS, because older versions use this method with other args. I'm not sure if that's an issue.

Reply:
Above might be a bit vague. Concretely, if we apply this fix to pytest-inmanta, the following would be compatible:
- new pytest-inmanta (with the fix): everything from 2020.4 up to (including) the next iso3, iso4 and OSS
- older pytest-inmanta (without the fix): everything from some unknown point before 2020.4 up to (including) the next iso3, iso4 and the latest OSS (so not the next OSS)
If we find this unacceptable I'm afraid we need to keep the method signature as is.